src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/LegacyPersonBuilder.kt

```diff
@@ -3,7 +3,7 @@ package uk.gov.justice.digital.hmpps.nomisprisonerapi.helper.builders
 import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.Person
 import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.PersonPhone
 
-class PersonBuilder(
+class LegacyPersonBuilder(
```

Comment:
renamed so it doesn't conflict with DSL builder - visit tests where never converted to use DSL